mono/mini/aot-runtime.h

```diff
@@ -74,6 +74,7 @@ typedef enum {
 	MONO_AOT_FILE_FLAG_SEPARATE_DATA = 64,
 	MONO_AOT_FILE_FLAG_EAGER_LOAD = 128,
 	MONO_AOT_FILE_FLAG_INTERP = 256,
+	MONO_AOT_FILE_FLAG_METHOD_TABLE_AS_DATA = 512
```

Comment:
Can you add a comment here, as to why not always data?
Is it historical, or must be code soetimes?
'cause ideally, minimize executable bytes, and data will tend to be smaller.
Also, can the data be position-independent ("RVA"s) instead of full pointers with relocations?
32bit RVA is exceedingly common on Windows, for example.